### ISSUE TITLE
Issue with unassigned members / PHP 7.4 (Typed property must not be accessed before initialization)

### ISSUE THREAD

**Issue by cbichis** [doctrine/doctrine-laminas-hydrator#25]
Hi,

Under PHP 7.4 there is issue with unassigned members.
```

File:

    /project/module/.........../IdTrait.php:18

Message:

    Typed property Supplier::$id must not be accessed before initialization

Stack trace:

    #0 /project/vendor/doctrine/doctrine-laminas-hydrator/src/DoctrineObject.php(219):Supplier->getId()
    #1 /project/vendor/doctrine/doctrine-laminas-hydrator/src/DoctrineObject.php(119): Doctrine\Laminas\Hydrator\DoctrineObject->extractByValue()
    #2 /project/vendor/laminas/laminas-form/src/Fieldset.php(659): Doctrine\Laminas\Hydrator\DoctrineObject->extract()
    #3 /project/vendor/laminas/laminas-form/src/Form.php(979): Laminas\Form\Fieldset->extract()
    #4 /project/vendor/laminas/laminas-form/src/Form.php(314): Laminas\Form\Form->extract()
```

```
/**
 * @ORM\Id
 * @ORM\GeneratedValue(strategy="IDENTITY")
 * @ORM\Column(name="id", type="integer", nullable=false, options={"unsigned"=true})
 * @Annotation\Exclude()
 */
protected int $id;
```

**Comment by driehle** [comment]
This is a duplicate of #15 and was already fixed with #20 and released as 2.1.1. @cbichis Could you please check if upgrading to 2.1.1 solves your issue?

**Comment by driehle** [comment]
Closing due to inactivity.
